lib/vision/index.js

```diff
@@ -1178,6 +1190,24 @@ Vision.findImages_ = function(images, callback) {
       return;
     }
 
+    // File is a URL.
+    if (image.indexOf('http') === 0) {
+      request({
+        method: 'GET',
+        uri: image,
+        encoding: 'base64'
+      }, function(err, resp, body) {
+        if (err) {
```

Comment:
It's just a style thing. Throughout our API, we always handle errors in the topmost portion of the code and return. That leaves the rest of the block available for un-indented handling of a successful response. Usually, that's much more code than just executing the callback in a slightly different way, but nonetheless, it would be breaking the pattern. IMO, it's better to be consistent than to have a super huge style guide that tries to define when to use what style.